src/validation/amountValidation.js

```diff
@@ -0,0 +1,25 @@
+import AmountType from '../types/AmountType';
+/* eslint-disable import/prefer-default-export */
+// returns string for helper text if amount is invalid
+export function invalidAmountFields(amountType, min, max) {
```

Comment:
I think you can fix the lint error like this:
```suggestion
// returns string for helper text if amount is invalid
export default function invalidAmountFields(amountType, min, max) {
```